File: backend/hhb_backend/alarm_evaluation/journal_entry_model.py
from dataclasses import dataclass, asdict


@dataclass
class JournalEntryModel:
    UUID: str
    Amount: int
    Date: int
    IsAutomaticallyReconciled: bool
    AgreementUuid: str
    BankTransactionUuid: str

    def to_dict(self):
        return asdict(self)

File: huishoudboekje_services/Core.CommunicationModels/JournalEntryModel/JournalEntryModel.cs
using Core.CommunicationModels.JournalEntryModel.Interfaces;

namespace Core.CommunicationModels.JournalEntryModel;

public class JournalEntryModel : IJournalEntryModel
{
  public string UUID { get; set; }

  public int Amount { get; set; }

  public long Date { get; set; }

  public bool IsAutomaticallyReconciled { get; set; }

  public string AgreementUuid { get; set; }

  public string BankTransactionUuid { get; set; }

  public string StatementUuid { get; set; }
}
